datahub-web-react/src/app/settings/SettingsPage.tsx

```diff
@@ -108,7 +112,7 @@ export const SettingsPage = () => {
                 <Menu
                     selectable={false}
                     mode="inline"
-                    style={{ width: 256, marginTop: 8 }}
+                    style={{ width: 256, marginTop: 8, overflowY: 'auto', overflowX: 'hidden' }}
```

Comment:
can we extract into a const object above?

Reply:
yes we can do it